src/cryo_challenge/_map_to_map/map_to_map_distance_matrix.py

```diff
@@ -12,22 +12,80 @@
 from ..data._validation.output_validators import MapToMapResultsValidator
 
 
-def vmap_distance(
-    maps_gt,
-    maps_submission,
-    map_to_map_distance,
-    chunk_size_gt=None,
-    chunk_size_submission=None,
-):
-    return torch.vmap(
-        lambda maps_gt: torch.vmap(
-            lambda maps_submission: map_to_map_distance(maps_gt, maps_submission),
-            chunk_size=chunk_size_submission,
-        )(maps_submission),
-        chunk_size=chunk_size_gt,
-    )(maps_gt)
-
-
+class MapToMapDistance:
```

Comment:
Classes should have an identifier that connects them to the option the user chooses in their config file. This can simply be in a docstring.